src/TreeView/TreeView.tsx

```diff
@@ -266,10 +276,31 @@ const Item: React.FC<TreeViewItemProps> = ({
               display: 'flex',
               alignItems: 'center',
               height: '100%',
-              px: 2
+              px: 2,
+              gap: 2
             }}
           >
-            {childrenWithoutSubTree}
+            <Slots>
+              {slots => (
+                // QUESTION: How should leading and trailing visuals impact the aria-label?
```

Comment:
Which aria-label are you talking about?

I don't think we'll need content that is accessible to assistive tech for _most_ trailing and leading visuals.

Reply:
I'm thinking about the `aria-label` for the element with `role=treeitem`.